contracts/core/base/LensMultiState.sol

```diff
@@ -39,11 +39,11 @@ abstract contract LensMultiState {
         return _state;
     }
 
-    function _setState(DataTypes.ProtocolState newState) internal {
-        DataTypes.ProtocolState prevState = _state;
-        _state = newState;
-        emit Events.StateSet(msg.sender, prevState, newState, block.timestamp);
-    }
+    // function _setState(DataTypes.ProtocolState newState) internal {
+        // DataTypes.ProtocolState prevState = _state;
+        // _state = newState;
+        // emit Events.StateSet(msg.sender, prevState, newState, block.timestamp);
+    // }
```

Comment:
Why are you commenting this function out? Is the `StateSet` event still needed?

Reply:
This logic is delegated to the `GeneralLib` now for code size.